includes/views/register.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
Register Sachen entfernen wenn nicht in diesem Branch fertiggestellt - halbfertiges soll **nie** in den master Branch kommen.